kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -53,19 +57,19 @@
 
     <table class="roster">
 
-      <caption class="visuallyhidden">Users</caption>
+      <caption class="visuallyhidden">{{$tr('users')}}</caption>
 
       <!-- Table Headers -->
       <thead v-if="usersMatchFilter">
         <tr>
-          <th class="col-header" scope="col"> Full Name </th>
+          <th class="col-header" scope="col"> {{$tr('fullName')}} </th>
           <th class="col-header" scope="col">
             <span class="role-header" aria-hidden="true">
-              Role
+              {{$tr('kind')}}
```

Comment:
I think `Kind` is less specific, whereas `Role` indicates it is what they can do in the system. Also, `kind` is inaccurate, because the same user can have a different role in different contexts.